src/BloomExe/BloomExe.csproj

```diff
@@ -115,6 +115,10 @@
       <HintPath>..\..\packages\MarkdownDeep.NET.1.5\lib\.NetFramework 3.5\MarkdownDeep.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="MarkdownSharp" Condition="'$(OS)'!='Windows_NT'">
```

Comment:
Can you comment on why this is special for Linux? Can a comment be added to the src for future reference?